packages/rocketchat-theme/client/imports/slider.css

```diff
@@ -0,0 +1,72 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+.range-slider {
+	margin: 0 0 0 0%;
+	width: 100%;
+}
+
+.range-slider__range {
+	-webkit-appearance: none;
```

Comment:
we already use autoprefixer on css build, so you can remove these browser prefixes.